developers/weaviate/api/graphql/additional-operators.md

```diff
@@ -125,14 +125,37 @@ The pagination implementation is an offset-based implementation, not a cursor-ba
 
 ## Autocut
 
-Starting with Weaviate `v1.20`, the `autocut` filter can be added as an argument to class objects retrieved via the `nearXXX`, `bm25` and `hybrid` operators. `autocut: <N>`, where N is an integer > 0, limits the number of results to those up to the Nth "jump"/"drop" in the distance/score from the query. For example, if the distances for six objects returned by `nearText` were `[0.1899, 0.1901, 0.191, 0.21, 0.215, 0.23]` then `autocut: 1` would return the first three objects, `autocut: 2` would return all but the last object, and `autocut: 3` would return all objects. Autocut is disabled by default, and can be disabled explicitly by setting its value to `0` or a negative number.
+:::info Added in `v1.20`
+:::
+
+The autocut filter limits results based on discontinuities in the result set. The filter looks for discontinuities, or jumps, in the result [scores](#score--explainscore). To use autocut, specify how many jumps there should be in your query. The query stops returning results after the specified number of jumps.
```

Comment:
Hey @daveatweaviate FYI for future reference, just be careful adding anchors like this, because the link checker is not able to check broken anchor links. (There are no anchors to #score--explainscore on this page)